content/zh/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -1814,10 +1813,12 @@ The CIDR to use for pod IP addresses, only used in standalone mode. In cluster m
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-The image whose network/IPC namespaces containers in each pod will use. This docker-specific flag only works when container-runtime is set to `docker`.
+ Specified image will not be pruned by the image garbage collector. When container-runtime is set to `docker`, all containers in each pod will use the network/ipc namespaces from this image. Other CRI implementations have their own configuration to set this image.
 -->
-指定基础设施镜像，Pod 内所有容器与其共享网络和 IPC 命名空间。
-仅当容器运行环境设置为 <code>docker</code> 时，此特定于 docker 的参数才有效。
+所指定的镜像不会被镜像垃圾收集器删除。
+当容器运行环境设置为 <code>docker</code> 时，各个 Pod 中的所有容器都会
```

Comment:
It's not related to the content in this PR. But this page contains a lot of code strings (enclosed in backtick) cannot be displayed in code format. Instead, the backtick is displayed directly. In your change, using the <code></code> tag fixes this problem for the `docker` string. 

How about I open another PR to update all other instances of code strings with <code></code> in the english version file?

Reply:
backticks are not properly processed because they are inside HTML tags. The markdown processor simply ignores them.